Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionExtractionWindow.cpp

```diff
@@ -284,6 +332,31 @@ namespace EMStudio
             }
 
             m_warningShowed = false;
+
+            m_rootMotionExtractionWidget->ClearInstances();
+            m_extractRootMotionCheck->setChecked(false);
+            if (numMotions == 1)
+            {
+                const EMotionFX::Motion* curMotion = selectionList.GetMotion(0);
+
+                const auto& rootMotionDataPtr = curMotion->GetRootMotionExtractionData();
```

Comment:
suggestion: `const auto*` as this looks like a pointer? (also same comment as before about scope)

Reply:
It's the smart_ptr, only name it this way because addInstance take a raw ptr to the data
calling it rootMotionDataSmartPtr is a bit overkill